.github/workflows/publish-helm-chart.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+      - chart-publish
+
+jobs:
+  package_chart:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: 3.10.2
+
+      - name: Package Chart
+        run: |
+          mkdir .chart-out
+          helm package chart/ --destination .chart-out
```

Comment:
Will this include `local.yaml` if it's been copied here, e.g. if someone is using `btrix`?

Reply:
Same question for `examples/` I suppose - I don't seen an option in `helm package` to exclude certain files

Reply:
yes, will include examples. this is designed to run only on CI so shouldn't have a `local.yaml` there